### PR TITLE
Update exceptions that are thrown

### PR DESCRIPTION
**Pull request by jdye64**
Closes: #503

### CODE REVIEW

File: dask_planner/src/sql.rs [hunk 1]
```diff
@@ -67,17 +73,22 @@ impl ContextProvider for DaskSQLContext {
...
+                        table::DaskTableSource::new(Arc::new(e)),
+                    )))),
+                    None => Err(DataFusionError::Plan(format!(
+                        "Table '{}.{}.{}' not found",
+                        reference.catalog, reference.schema, reference.table
+                    ))),
+                }
             }
+            None => Err(DataFusionError::Plan(format!(
+                "Unable to located Schema: '{}.{}'",
```
Comment (andygrove) [inline dask_planner/src/sql.rs:89]:
```suggestion
                "Unable to locate Schema: '{}.{}'",
```

File: dask_planner/src/sql.rs [hunk 2]
```diff
@@ -159,13 +171,10 @@ impl DaskSQLContext {
     ) -> PyResult<logical::PyLogicalPlan> {
         let planner = SqlToRel::new(self);
         match planner.statement_to_plan(statement.statement) {
```
Comment (andygrove) [inline dask_planner/src/sql.rs:173]: nit: we can make this section more idiomatic using `map` and `map_err`.

``` rust
planner
            .statement_to_plan(statement.statement)
            .map(|k| logical::PyLogicalPlan {
                original_plan: k,
                current_node: None,
            })
            .map_err(|e| PyErr::new::<ParsingException, _>(format!("{}", e)))
```

**Review by andygrove — APPROVED** [review]
Rust changes LGTM

**Review by ayushdg — APPROVED** [review]
Python changes lgtm

**Pull request merged**